content/00.front-matter.md

```diff
@@ -20,9 +20,11 @@ was automatically generated
 {% if manubot.ci_source is defined -%}
 from [{{manubot.ci_source.repo_slug}}@{{manubot.ci_source.commit | truncate(length=7, end='', leeway=0)}}](https://github.com/{{manubot.ci_source.repo_slug}}/tree/{{manubot.ci_source.commit}})
 {% endif -%}
-on {{manubot.date}}.
+on {{manubot.generated_date_long}}.
 </em></small>
 
+Published: {{manubot.date_long}}
```

Comment:
I don't have too strong a preference of where we display this.  The main point is to show which template variable to use.  It might make sense to make it commented or include it as part of the existing commented block in lines 7-9.  Most manuscripts won't to display this date until a major version is finished.

Reply:
I made the display conditional in https://github.com/manubot/rootstock/pull/478/commits/10f1616ef79d684dc2839cdc65b1d79fcb83a1fe... only display publication date if its different than the generated date.

Placing it with "A DOI-citeable version of this manuscript is available at..." but that text gets centered and it didn't quite look right to me.

Reply:
👍 that works for me